a3p-integration/proposals/c:names-by-addr/post.test.js

```diff
@@ -0,0 +1,124 @@
+// @ts-check
+import test from 'ava';
+import { createRequire } from 'module';
+import { execFileSync } from 'child_process';
+import tmp from 'tmp';
+import { readFile as readFileAmbient, writeFile } from 'fs/promises';
+import { getIncarnation } from '@agoric/synthetic-chain/src/lib/vat-status.js';
+import {
+  voteLatestProposalAndWait,
+  waitForBlock,
+} from '@agoric/synthetic-chain/src/lib/commonUpgradeHelpers.js';
+import { agoric } from '@agoric/synthetic-chain/src/lib/cliHelper.js';
+import { makeAgd } from '@agoric/synthetic-chain/src/lib/agd-lib.js';
+import * as env from '@agoric/synthetic-chain/src/lib/constants.js';
+
+const nodeRequire = createRequire(import.meta.url);
+const asset = {
+  sendScript: nodeRequire.resolve('./send-script.js'),
+};
+
+// #region copy-pasta from @agoric/synthetic-chain
+
+// XXX import from '@agoric/synthetic-chain/src/lib/core-eval-support.js';
+
+/**
+ * @param { Record<string, string>} record - e.g. { color: 'blue' }
+ * @returns {string[]} e.g. ['--color', 'blue']
+ */
+export const flags = record => {
+  return Object.entries(record)
+    .map(([k, v]) => [`--${k}`, v])
+    .flat();
+};
+
+export const txAbbr = tx => {
+  const { txhash, code, height, gas_used } = tx;
+  return { txhash, code, height, gas_used };
+};
+
+// XXX overlaps with passCoreEvalProposal from synthetic chain
```

Comment:
why not use it?
https://github.com/Agoric/agoric-3-proposals/blob/6e5fec521b33a3479ba9a6fd198b9313f49c5a7f/packages/synthetic-chain/src/lib/core-eval.ts#L86-L89

Reply:
hm... the reason has faded. something about staticConfig? I'll have to look again.